app/models/app.model.js

```diff
@@ -36,8 +37,40 @@ exports.selectArticles = () => {
         FROM articles
         LEFT OUTER JOIN comments ON comments.article_id = articles.article_id
         GROUP BY articles.article_id
-        ORDER BY articles.created_at DESC;`)
+        ORDER BY articles.created_at DESC;`
+    )
     .then(({rows}) => {
         return rows;
     })
-}
\ No newline at end of file
+}
+
+exports.selectCommentsByArticleId = (article_id) => {
+    return db
+    .query(`SELECT * FROM articles WHERE article_id = $1`,[article_id])
+    .then(({rows}) => {
+        if(rows.length === 0){
+            return Promise.reject({
+                status: 404,
+                msg: `No article found under article_id ${article_id}.`
+            })
+        } else{
+            return db
+            .query(
+                `SELECT 
+                comments.comment_id,
+                comments.votes,
+                comments.created_at,
+                comments.author,
+                comments.body,
+                comments.article_id
+                FROM comments
+                WHERE comments.article_id = $1
+                ORDER BY created_at DESC`,
+                [article_id]
```

Comment:
You should be able to merge both of these sql queries into one. If the article doesn't exist, you can just promise.reject the single query 👍